fpm/src/fpm_command_line.f90

```diff
@@ -1,16 +1,17 @@
 module fpm_command_line
-    use fpm_environment, only: get_os_type, OS_LINUX, OS_MACOS, OS_WINDOWS
-
+    use :: fpm_environment, only: get_os_type, &
```

Comment:
Probably best to be consistent with all `use` statements now. (no colons)

Reply:
(this is most likely affected by the same bug but compiles in the right order by happenchance)